detox/test/ios/AppDelegate.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+import React
+
+// MARK: - App Delegate
+@UIApplicationMain
+@objc(AppDelegate)
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    var moduleName: String = "example"
+    var initialProps: [String: Any] = [:]
+    private var screenManager: NativeScreenManaging?
+
+    // MARK: - UIApplicationDelegate Methods
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+    ) -> Bool {
+        setupReactNative(with: launchOptions)
+        setupNotifications()
+        setupScreenManager()
+        setupApplicationStateObservers()
+        setupShakeDetection()
+
+        return true
+    }
+
+    // MARK: - Setup Methods
+
+    private func setupReactNative(with launchOptions: [UIApplication.LaunchOptionsKey: Any]?) {
+        let bridge = RCTBridge(delegate: self, launchOptions: launchOptions)
```

Comment:
Is this fine for New Arch Bridgeless mode which is default?

Reply:
Good point. I'm using the old pattern here.. I might have overlooked it and need to migrate to the `RCTAppDelegate` class, which should manage both bridgeless mode and the new architecture mode. I'm checking it now.